cmd/root.go

```diff
@@ -110,5 +113,6 @@ func runSignalWrapper(cmd *cobra.Command, args []string) error {
 	default:
 		cmd.PrintErrf("The proxy has encountered a terminal error: %v\n", err)
 	}
+	cmd.SilenceUsage = true
```

Comment:
So my hot take is that probably needs to happen in `newProxyClient()`, after we've parsed the instance connection string. Otherwise things like mount errors will cause the usage to print when maybe the don't need too. Thoughts?